### PR TITLE
Add a note about using raw data attributes as a prefered way

### PR DESCRIPTION
**Pull request by bocharsky-bw**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| Issues        | no 
| License       | MIT

Now with the Stimulus plugin autocomplete in PhpStorm it's better to use raw data attributes instead of built-in functions/filters. Normal Stimulus code is more readable and cleaner IMO.

### CODE REVIEW

**Comment by weaverryan** [comment]
I need to check the PR still, but I’m also preferring the direct attributes these days.

File: src/StimulusBundle/doc/index.rst [hunk 1]
```diff
@@ -65,6 +65,11 @@ Use the Twig functions from this bundle to activate your controllers:
         ...
     </div>
 
+    <!-- would render -->
+    <div data-controller="hello">
+       ...
+    </div>
```
Comment (weaverryan) [inline src/StimulusBundle/doc/index.rst:79]: We should update a bit more, starting with the "Use the Twig functions..." line. Something like:

> Then, activate the controller in your HTML:

(show the raw `data-controller` example).

Then after, something like:

Optionally, this bundle has a Twig function to render the attribute:

(show the `stimulus_controller()`)

File: src/StimulusBundle/doc/index.rst [hunk 2]
```diff
@@ -159,6 +164,13 @@ Stimulus Twig Helpers
 This bundle adds 3 Twig functions/filters to help add Stimulus controllers,
 actions & targets in your templates.
 
+.. note::
+
+    Though this bundle provides these helpful functions/filters, it's recommended to
+    use raw data attributes instead. If you use PhpStorm - you may want to install
```
Comment (weaverryan) [inline src/StimulusBundle/doc/index.rst]: attributes instead, as they're simple enough. If you use...

File: src/StimulusBundle/doc/index.rst [hunk 3]
```diff
@@ -159,6 +164,13 @@ Stimulus Twig Helpers
...
+    [Stimulus plugin](https://plugins.jetbrains.com/plugin/18940-stimulus) that will
+    bring you nice autocompletion for them.
```
Comment (weaverryan) [inline src/StimulusBundle/doc/index.rst]: Stimulus plugin to get nice auto-completion for the attributes

**Comment by bocharsky-bw** [comment]
@weaverryan done! Thanks for the review!

File: src/StimulusBundle/doc/index.rst [hunk 4]
```diff
@@ -159,6 +172,14 @@ Stimulus Twig Helpers
 This bundle adds 3 Twig functions/filters to help add Stimulus controllers,
 actions & targets in your templates.
 
+.. note::
+
+    Though this bundle provides these helpful Twig functions/filters, it's
+    recommended to use raw data attributes instead, as they're simple enough.
+    If you use PhpStorm IDE - you may want to install
```
Comment (94noni) [inline src/StimulusBundle/doc/index.rst:182]: I would add this phrase as a tip bloc :)

File: src/StimulusBundle/doc/index.rst [hunk 5]
```diff
@@ -159,6 +172,14 @@ Stimulus Twig Helpers
 This bundle adds 3 Twig functions/filters to help add Stimulus controllers,
 actions & targets in your templates.
 
+.. note::
+
+    Though this bundle provides these helpful Twig functions/filters, it's
+    recommended to use raw data attributes instead, as they're simple enough.
```
Comment (94noni) [inline src/StimulusBundle/doc/index.rst]: Perhaps replace « simple enough » with « straight forward » ?

**Review by 94noni — APPROVED** [review]
I second this, it should be the recommended way on doc :)

**Comment by bocharsky-bw** [comment]
@94noni thank you for your review, done!

**Comment by weaverryan** [comment]
Thanks Victor!

**Pull request merged**